CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+2.0.1 - 2025-04-10
+------------------
+
+- Fix compatibility with minify-html 0.16.0 and newer
+
```

Comment:
Because [AutoPub](https://justinmayer.com/projects/autopub/) automatically updates the change-log, there is no need to do so manually. And in fact, doing so manually would create redundant change-log entries, so please move this bullet point to a `RELEASE.md` file with `Release type: patch` and add that file to this PR. ([Example](https://github.com/pelican-plugins/minify/blob/4fa25d5078bfd6d969c19efd47f6cbb145039e95/RELEASE.md?plain=1))